include/bluetooth/conn.h

```diff
@@ -63,6 +63,86 @@ struct bt_le_conn_param {
 						  BT_GAP_INIT_CONN_INT_MAX, \
 						  0, 400)
 
+/** Connection PHY information for LE connections */
+struct bt_conn_le_phy_info {
+	u8_t tx_phy; /** Connection transmit PHY */
+	u8_t rx_phy; /** Connection receive PHY */
+};
+
+/** Preferred PHY parameters for LE connections */
+struct bt_conn_le_phy_param {
+	u8_t pref_tx_phy; /** Bitmask of preferred transmit PHY */
+	u8_t pref_rx_phy; /** Bitmask of preferred receive PHY */
+};
+
+/** Helper to declare PHY parameters inline
+ *
+ * @param pref_tx_phy Bitmask of preferred transmit PHY.
+ * @param pref_rx_phy Bitmask of preferred receive PHY.
+ */
+#define BT_CONN_LE_PHY_PARAM(pref_tx_phy, pref_rx_phy) \
+	((struct bt_conn_le_data_len_param[]) { {      \
+		.pref_tx_phy = (pref_tx_phy),          \
+		.pref_rx_rx = (pref_rx_phy),           \
+	 } })
+
+/** Only LE 1M PHY */
+#define BT_CONN_LE_PHY_PARAM_1M BT_CONN_LE_PHY_PARAM(BT_GAP_LE_PHY_1M, \
+						     BT_GAP_LE_PHY_1M)
+
+/** Only LE 2M PHY */
+#define BT_CONN_LE_PHY_PARAM_2M BT_CONN_LE_PHY_PARAM(BT_GAP_LE_PHY_2M, \
+						     BT_GAP_LE_PHY_2M)
+
+/** Only LE Coded PHY. */
+#define BT_CONN_LE_PHY_PARAM_CODED BT_CONN_LE_PHY_PARAM(BT_GAP_LE_PHY_CODED, \
+							BT_GAP_LE_PHY_CODED)
+
+/** All LE PHYs. */
+#define BT_CONN_LE_PHY_PARAM_ALL BT_CONN_LE_PHY_PARAM(BT_GAP_LE_PHY_1M |   \
+						      BT_GAP_LE_PHY_2M |   \
+						      BT_GAP_LE_PHY_CODED, \
+						      BT_GAP_LE_PHY_1M |   \
+						      BT_GAP_LE_PHY_2M |   \
+						      BT_GAP_LE_PHY_CODED)
+
+/** Connection data length information for LE connections */
+struct bt_conn_le_data_len_info {
+	u16_t tx_max_len;  /** Connection transmit max data length in bytes. */
```

Comment:
Suggested rephrasing:
```suggestion
	u16_t tx_max_len;  /** Maximum Link Layer transmission payload size in bytes. */
```
Similar for others